src/components/Drop/Drop.test.js

```diff
@@ -112,7 +112,7 @@ describe('Composed', () => {
 describe('Trigger', () => {
   test('Can render', () => {
     const TestComponent = Drop()(ContentComponent)
-    const wrapper = shallow(<TestComponent isOpen trigger={trigger} />)
+    const wrapper = shallow(<TestComponent isOpen trigger={trigger} />).dive()
```

Comment:
TIL: didn't know `dive` even existed, I don't write enzyme tests anymore but good to know for when updating existing ones it might make things easier, thanks!

Reply:
Funny story (well, funny now, it wasn't much fun back then :smile:). We've been using only `shallow` at some point and with a lot of refactoring or adding new features, tons of tests were failing, because of not rendered elements. It started to look like `shallow().dive().dive().dive().dive()...`, so we've created function `diveUpTo(selector)`, which was doing `dive()` in loop until selector was found. That wasn't great idea, but at least since then I am not using `shallow` anymore :smile: